packages/table/src/_docs.scss

```diff
@@ -124,6 +124,30 @@ Styleguide components.table-js.example-editable
 */
 
 /*
+Loading states
+
+When fetching or updating data, it may be desirable to show a loading state.
```

Comment:
add:

The Table components provide fine-grain loading control of loading row headers, column headers, or individual cells.